docs/fix/dry-buckets.rst

```diff
@@ -19,6 +19,7 @@ manually.
 Usage
 -----
 
-::
-
-    fix/dry-buckets
+``fix/dry-buckets``
```

Comment:
```suggestion
::

    fix/dry-buckets [<options>]

Examples
--------

``fix/dry-buckets``
```

Reply:
this would normally have an "Options" section as well, but I don't think it's strictly required here as there's only one